.devcontainer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:lts-alpine
+
+ENV NODE_ENV production yarn build
+
+ENV THELOUNGE_HOME "~/.thelounge"
+VOLUME "${THELOUNGE_HOME}"
+
+ENV PORT 9000
+EXPOSE ${PORT}
+
+RUN apk --update --no-cache --virtual add git openssh
```

Comment:
Missing python, build-essentials or whatever it's named in musl land.
You'll have trouble getting the sqlite3 deb otherwise.

Reply:
sorry for being dense but afaict its building and querying the sqlite db with `["sqlite", "text"]` as my config setting :thinking:

Reply:
On the master branch?

We pin a specific commit and the way it works is that sqlite3 tries to find a binary blob with the current Version/Arch/nodeAbi/platform and if it can't find that it falls back to building it.

If it still works if you don't actually have a compiler that means that you've pulled the blob. Which surprises me a tad.
Then it would look like it's only fetching the latest release version of the blob, no matter that it could've changed in the repo.

And considering that they changed the compilation flags but didn't bump any versions afaict and hence can't ID a binary blob with those mods... well.

https://github.com/mapbox/node-sqlite3/commit/918052b538b0effe6c4a44c74a16b2749c08a0d2

Reply:
thanks for the explanation. ive added `sqlite` to the image: https://github.com/thelounge/thelounge/pull/4514/commits/a90c59f01037c074c3841d84c3e400ad35d1ea1e

Reply:
undid the previous commit.